Description: Add Keywords, and fix Exec & Icon in desktop file.
Author: Bas Couwenberg <sebastic@xs4all.nl>

--- a/gui/icons/grass.desktop
+++ b/gui/icons/grass.desktop
@@ -47,7 +47,7 @@ Comment[tr]=Coğrafi bilgi sistemi
 Comment[uk]=Геоінформаційна система
 Comment[uz]=Geografik axborot tizimi
 Categories=Education;Science;Geoscience;Geography;
-Exec=grass7
-Icon=grass
+Exec=grass70
+Icon=grass70
 Terminal=true
 Keywords=wms;wfs;ogc;osgeo
